doc/source/serve/getting_started.md

```diff
@@ -65,339 +71,279 @@ PyTorch, and Tensorflow for more info and examples:
-define a function called `router` that takes in a Starlette `request`
-object [^f2]:
+With that, we can run our model on Ray Serve!
+Here's the full Ray Serve script that we built:
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_deployment.py
-:end-before: __router_end__
+```{literalinclude} ../serve/doc_code/getting_started/model_deployment_full.py
+:start-after: __deployment_full_start__
+:end-before: __deployment_full_end__
 :language: python
 :linenos: true
-:start-after: __router_start__
 ```
 
-- In line 1, we add the decorator `@serve.deployment`
-  to the `router` function to turn the function into a Serve `Deployment`
-  object.
-- In line 3, `router` uses the `"txt"` query parameter in the `request`
-  to get the article text to summarize.
-- In line 4, it then passes this article text into the `summarize` function
-  and returns the value.
-
-:::{note}
-Lines 3 and 4 define our HTTP request schema. The HTTP requests sent to this
-endpoint must have a `"txt"` query parameter that contains a string.
-In general, you can accept HTTP data using query parameters or the
-request body. Additionally, you can add other Serve deployments with
-different names to create more endpoints that can accept different schemas.
-For more complex validation, you can also use FastAPI (see
-{ref}`serve-fastapi-http` for more info).
-:::
+We can run our script with the `serve run` CLI command. This command takes in an import path
+to our deployment formatted as `module:bound_deployment`. Make sure to run the command from a directory containing a local copy of this script, so it can find the bound deployment:
 
-:::{tip}
-This routing function's name doesn't have to be `router`.
-It can be any function name as long as the corresponding name is present in
-the HTTP request. If you want the function name to be different than the name
-in the HTTP request, you can add the `name` keyword parameter to the
-`@serve.deployment` decorator to specify the name sent in the HTTP request.
-
-For example, if the decorator is `@serve.deployment(name="responder")` and
-the function signature is `def request_manager(request)`, the HTTP request
-should use `responder`, not `request_manager`. If no `name` is passed
-into `@serve.deployment`, the `request` uses the function's name by
-default. For example, if the decorator is `@serve.deployment` and the
-function's signature is `def manager(request)`, the HTTP request should use
-`manager`.
-:::
+```console
+$ serve run serve_deployment:translator
+```
+
+This command will start running `Translator` and then block. It can be killed with `ctrl-C` in the terminal.
 
-Since `@serve.deployment` makes `router` a `Deployment` object, it can be
-deployed using `router.deploy()`:
+## Testing Ray Serve Deployments
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_deployment.py
-:end-before: __router_deploy_end__
+We can now test our model over HTTP. It can be reached at `http://127.0.0.1:8000/`
```

Comment:
Missing a period but looks awkward to add a period after a URL.  Maybe:

It can be reached at the following URL:

```
http://127.0.0.1:800
```

Reply:
I replaced the sentence with your suggestion.